hieradata_aws/class/staging/release_db_admin.yaml

```diff
@@ -11,15 +11,15 @@ govuk_env_sync::tasks:
     temppath: "/tmp/release_production"
     url: "govuk-production-database-backups"
     path: "mysql"
-  # "push_release_production_daily":
-  #   ensure: "present"
-  #   hour: "5"
-  #   minute: "0"
-  #   action: "push"
-  #   dbms: "mysql"
-  #   storagebackend: "s3"
-  #   database: "release_production"
-  #   database_hostname: "release-mysql"
-  #   temppath: "/tmp/release_production"
-  #   url: "govuk-staging-database-backups"
-  #   path: "release-mysql"
+  "push_release_production_daily":
```

Comment:
I think you meant to edit the `search_admin_db_admin.yaml` file here 😄.

Reply:
yea sorry good spot!